assets/party/identifier-lookup.js

```diff
@@ -24,9 +24,10 @@ $(function () {
           $('#party_form_address_side').val('')
           $('#party_form_address_postalCode').val('')
           $('#party_form_address_city').val('')
+          $('#party_form_isUnderAddressProtection').prop('checked', false)
 
           // Indicate that identifier was not found
-          $($identificationLookupButton).removeClass().addClass('btn-danger btn mt-2 identification-lookup')
+          $($identificationLookupButton).removeClass().addClass('btn-danger btn identification-lookup')
```

Comment:
You could consider using https://api.jquery.com/removeclass/#removeClass-function to remove only the class(es) starting with `btn-` here to not have to add all classes on the element.

Reply:
Updated to only remove class starting with `btn-` with use of regular expression!